packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -1,5 +1,4 @@
 const path = require("path");
-const CleanWebpackPlugin = require("clean-webpack-plugin").CleanWebpackPlugin;
```

Comment:
clean-webpack-plugin works with webpack 5 javascript projects, but throws errors in typescript project. See https://github.com/johnagan/clean-webpack-plugin/issues/188